server/routes/board.js

```diff
@@ -1,16 +1,23 @@
-const express = require("express");
-const router = express.Router();
+const router = require("express").Router();
 const protect = require("../middleware/auth");
 const {
-  getBoards,
   createBoard,
   getBoard,
-  updateBoard,
+  updateBoardName,
+  deleteBoard,
+  swapBoardColumns,
+  createBoardColumn,
+  deleteBoardColumn,
 } = require("../controllers/board");
 
-router.route("/").get(protect, getBoards);
-router.route("/").post(protect, createBoard);
-router.route("/:id").get(protect, getBoard);
-router.route("/:id").patch(protect, updateBoard);
+router.use(protect);
+
+router.route("/").post(createBoard);
+router.route("/:id").get(getBoard);
+router.route("/:id").patch(updateBoardName);
+router.route("/:id/addColumn").patch(createBoardColumn);
```

Comment:
This should be a `POST` request and not a `PATCH` because we are adding a new column to the database. `PATCH` requests are more for editing resources that already exist.

Reply:
Changed to POST